test/getiBFT_spec.rb

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env rspec
+require_relative '../src/modules/IscsiClientLib'
+
+describe Yast::IscsiClientLibClass do
+
+  before :each do
+    @iscsilib = Yast::IscsiClientLibClass.new
+    @iscsilib.main() 
+  end
```

Comment:
rspec allows for such purpose use of `subject` definition. See https://www.relishapp.com/rspec/rspec-core/v/2-11/docs/subject/explicit-subject